src/Microsoft.ML.StandardLearners/Standard/MultiClass/doc.xml

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+
+    <member name="MultiClassNaiveBayesTrainer">
+      <summary>
+        Trains a multiclass Naive Bayes predictor that supports binary feature values.
+      </summary>
+      <remarks>
+        <a href ='https://en.wikipedia.org/wiki/Naive_Bayes_classifier'>Naive Bayes</a> is a probabilistic classifier that can be used for multiclass problems.
+        Using Bayes' theorem, the conditional probability for a sample belonging to a class can be calculated based on the sample count for each feature combination groups.
+        However, Naive Bayes Classifier is feasible only if the number of features and the values each feature can take is relatively small.
+        It assumes independence among the presence of features in a class even though they may be dependent on each other.
+        This multi-class trainer accepts binary feature values of type float, i.e., feature values are either true or false.
+        Specifically a feature value greater than zero is treated as true.
+        These learner will request normalization from the data pipeline if the
+        classifier indicates it would benefit from it. Note that even if the
+        classifier indicates that it does not need caching, OVA will always
+        request caching, as it will be performing multiple passes over the data set.
+      </remarks>
+      <seealso cref='Microsoft.ML.Trainers.LogisticRegressionClassifier'></seealso>
+      <seealso cref='Microsoft.ML.Trainers.LightGbmClassifier'></seealso>
+      <seealso cref='Microsoft.ML.Trainers.StochasticDualCoordinateAscentClassifier'></seealso>
+      <seealso cref='Microsoft.ML.Models.OneVersusAll'></seealso>
+      <example>
+        <code>
+          pipeline.Add(new NaiveBayesClassifier(){ NormalizeFeatures = NormalizeOption.Auto, Caching = CachingOptions.Memory });
+        </code>
+      </example>
+    </member>
+
+    <member name="OVA">
+      <summary>
+        In this strategy, a binary classification algorithm is used to train one classifier for each class, which distinguishes that class from all other classes.
+        Prediction is then performed by running these binary classifiers, and choosing the prediction with the highest confidence score.
+      </summary>
+      <remarks>
+        <para>This algorithm can be treated as a wrapper for all the binary classifiers in ML.NET. 
+        A few binary classifiers already have implementation for multi-class problems, 
+        thus users can choose either one depending on the context. 
+        </para>
+        <para>
+          The OVA version of a binary classifier, such as wrapping a LightGbmBinaryClassifier ,
+          can be different from LightGbmClassifier, which develops a multi-class classifier directly. 
+        </para>
+      </remarks>
+      <seealso cref='Microsoft.ML.Trainers.LogisticRegressionClassifier'>LogisticRegressionClassifier</seealso>
```

Comment:
><seealso cref='Microsoft.ML.Trainers.LogisticRegressionClassifier'>LogisticRegressionClassifier</seealso> [](start = 6, length = 105)

My expectation with `<seealso` tags is that they take the form `<seealso cref="Foo"/>` not `<seealso cref="Foo">Foo</seealso>`. That latter form, even if it were supported, I would discourage us from using because it will *not* track renames of the elements.

Reply:
Switched to this when i needed the absolute name, because all the namespace repetition looked lousy. Agree that the renaming is more important, thought. switching back. 

---
In reply to: [202460733](https://github.com/dotnet/machinelearning/pull/529#discussion_r202460733) [](ancestors = 202460733)